routes/users.js

```diff
@@ -135,4 +134,35 @@ router.get("/:id", async (req, res, next) => {
   }
 });
 
+router.patch("/:id", async (req, res, next) => {
+  const { nickname, imageUrl } = req.body;
+  const { auth: token } = req.cookies;
+
+  const { _id: userId } = jwt.decode(token);
```

Comment:
params로 넘어오는 id와 디코드 된 토큰의 id를 비교해서 

수정하려는 사용자 정보가 수정을 요청한 사용자와 동일한지 확인하는 로직이 필요할 것 같습니다!

Reply:
```suggestion
router.patch("/:id", async (req, res, next) => {
  const { id : userId } = req.params;
  const { nickname, imageUrl } = req.body;
  const { auth: token } = req.cookies;

  const { _id: decodedId } = jwt.decode(token);

  try {
    if (userId !== decodedId) {
      throw createError(403, NO_AUTHORITY_TO_ACCESS);
    }
```

이렇게 수정했는데 말씀하신 의도와 같을까요!